webpack/base.js

```diff
@@ -32,6 +32,8 @@ module.exports = {
     new ExtractTextPlugin('style.css', {
       allChunks: true
     }),
+    new webpack.NoErrorsPlugin(),
+    new webpack.optimize.OccurenceOrderPlugin(),
```

Comment:
We can't use -p since it would conflict with our custom minimization configuration in production.